README.md

```diff
@@ -16,6 +16,9 @@ To run in dev mode:
 To launch Postgraphile server (in a new Terminal process):  
 `yarn pg`
 
+To launch Postgraphile server as graphile_user (in a new Terminal process), this will enforce row level permissions:  
+`yarn pg_permissions`
+
 To launch in dev mode _with_ Postgraphile server:  
 `yarn dev_pg`
 
```

Comment:
```suggestion

To launch in dev mode _with_ Postgraphile server with row-level permissions:  
`yarn dev_pg_permissions`
```
See my later comment for adding this additional script (in package.json)

Reply:
0ef20cc3de12ee877e283e4c71a77deebfa0420e